.github/workflows/wheels-linux.yml

```diff
@@ -0,0 +1,55 @@
+# Build, test and publish wheels to PyPI
+# Build and test run on every push
+# Publish runs only on a tag push
+# Note: This does not publish sdist
+name: Wheels (Linux)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheels on Linux
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - uses: actions/setup-python@v2
+      name: Install Python
+      with:
+        python-version: 3.8
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.4.2
+
+    - name: Build wheels
+      env:
+        # Skip building on Python 2.7 and PyPy
+        CIBW_SKIP: cp27-* pp*
+        CIBW_BEFORE_TEST: pip install -r tests/requirements.txt
+        CIBW_TEST_COMMAND: pytest {project}/tests
+      run: |
+        python -m cibuildwheel --output-dir dist
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: ./dist
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/')
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: dist
+          path: dist/
+      - name: Publish to PyPI
```

Comment:
If we want to leverage this later to publish to pypi, where do I set the `__token__` etc? Do you have any documentation on this?
Thanks!

Reply:
You should be able to add here: https://github.com/bobfang1992/pytomlpp/settings/secrets

Get the token from your PyPI account, you can create one just for the repo ideally.

Reply:
Great

Reply:
Good thanks for the clarification.

Reply:
Action documentation: https://github.com/pypa/gh-action-pypi-publish